### ISSUE TITLE
Documentation ambiguous regrading memory utilization 

### ISSUE THREAD

**Issue by DP19** [istio/istio#42843]
Unsure of where this should be directed; but if there's a better place for documentation issues please let me know.

While we were looking to set resource requirements for a cluster that gets around 150k requests per second we found the [Performance and Scalability](https://istio.io/latest/docs/ops/deployment/performance-and-scalability) page in the docs.

On this page theres two sections that contradict each other.

The [Performance Summary](https://istio.io/latest/docs/ops/deployment/performance-and-scalability/#performance-summary-for-istio-hahahugoshortcode-s0-hbhb) section states that:

```The Envoy proxy uses 0.35 vCPU and 40 MB memory per 1000 requests per second going through the proxy.```

But further down in the [CPU and Memory](https://istio.io/latest/docs/ops/deployment/performance-and-scalability/#cpu-and-memory) section it states:

```Since the proxy normally doesn’t buffer the data passing through, request rate doesn’t affect the memory consumption.```

I'd like to understand the differences here and update the docs to reflect this to be less ambiguous.

Is it that envoy by itself uses 40mb per 1000 requests and istio doesn't add any overhead? Is it referencing two different components, the data plane vs proxies?

**Comment by kyessenov** [comment]
It's not contradictory. The same 40MB are reused for subsequent requests. What that means is that once Envoy is in a steady state, you should not see memory growing. The steady state memory does depend on the configuration and QPS.

**Comment by dwj300** [comment]
@kyessenov can you elaborate on the CPU statement(s)? On this same page, I see `The Envoy proxy uses 0.35 vCPU and 40 MB memory per 1000 requests per second going through the proxy.` and `Since the sidecar proxy performs additional work on the data path, it consumes CPU and memory. In Istio 1.16.2, a proxy consumes about 0.5 vCPU per 1000 requests per second.`

**Comment by kyessenov** [comment]
Sorry, I wasn't paying attention to that page in awhile. First, it is not right to keep posting the same performance numbers without re-running experiments (cc @ericvn @howardjohn ). 1.16 came out in 2022, so we should confirm they hold true today, as well as resolve the contradictory statements.

For any performance evaluation, it's best to look up a third party or validate the performance in your setup. We can provide help in tuning and reaching optimal performance for Envoy.

**Comment by DP19** [comment]
Thanks for the info; we're running Anthos Service Mesh in GKE v1.13, which is branded istio and we roughly see those metrics hold true

**Comment by keithmattix** [comment]
@johnma14 I believe we did performance tests last release; what's the latest on getting them published?

**Comment by johnma14** [comment]
We did run the performance tests against the 1.17 release (we had someone from Intel help run these tests) but there were some discrepancy in the results we saw, so we need to re-run these tests. The person who was helping us rerun these tests no longer works on Istio so we need to figure out how to setup the env and what/how to rerun these tests again. More details are here: https://github.com/istio/istio/issues/44009. @kfaseela fyi.
